docs/azure-multiple-vmss.md

```diff
@@ -27,7 +27,7 @@ Muchos provides a [sample file](../conf/azure_multiple_vmss_vars.yml.example) wh
 | `azure_disk_device_pattern`| Optional | If not specified, the corresponding `azure_disk_device_pattern` value from the `azure` section in [muchos.props](../conf/muchos.props.example) is used | This is a device name wildcard pattern used (internally) in conjunction with `azure_disk_device_path` to enumerate attached SCSI or NVME disks to use for persistent local storage |
 | `mount_root`| Optional | If not specified, the corresponding `mount_root` value from the `azure` section in [muchos.props](../conf/muchos.props.example) is used | This is the folder in the file system where the persistent disks are mounted |
 | `data_disk_count`| Required | - | An integer value which specifies the number of persistent (managed) data disks to be attached to each VM in the VMSS. It can be 0 in specific cases - see [notes on using ephemeral storage](./azure-ephemeral-disks.md) for details |
-| `disk_sku`| Required | - | Can be either Standard_LRS (for HDD) or Premium_LRS (for Premium SSD). At this time, we have not tested the use of Standard SSD or UltraSSD with Muchos |
+| `data_disk_sku`| Required | - | Can be either Standard_LRS (for HDD) or Premium_LRS (for Premium SSD). At this time, we have not tested the use of Standard SSD or UltraSSD with Muchos |
```

Comment:
Should osdisk_sku also have an entry in this table?

Reply:
Good question - `osdisk_sku` is actually just a local variable in the create_vmss*.yml playbooks. It is used for convenience to represent the jinja2 expression which decides whether (or not) the OS disk can be a Premium SSD, based on the capability of the VM to support such disks. If this clarifies, please resolve the comment post which I will merge.